tests/Unit/DataStructures/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ set(LIBRARY_SOURCES
   Test_StripeIterator.cpp
   Test_Variables.cpp
   Test_VectorImplHelper.cpp
+  Test_VectorImpl.cpp
```

Comment:
We've been sorting these ignoring file extension (see `src/CMakeLists.txt` with `Domain.cpp`)
```
  Test_VectorImpl.cpp
  Test_VectorImplHelper.cpp
```